src/components/paymentMethod/CustomRadioButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+interface CustomRadioButtonProperties {
+  name: string;
+  value: string;
+  checked?: boolean;
+}
+
+const CustomRadioButton: React.FC<CustomRadioButtonProperties> = ({
+  name,
+  value,
+  checked,
+}) => {
+  return (
+    <label className="inline-flex items-center space-x-2">
+      <input
```

Comment:
shadcn already has a radio group component. dont reinvent the wheel
https://ui.shadcn.com/docs/components/radio-group

Reply:
i need to customize the color of the radio button to fit the figma design
shadcn doesnt alllow one to customize color of radio button

Reply:
it actually does @Goldyn19. thats what the className prop is for. and if it doesnt, you cutomize the radio component to do so. The problem with creating your own components from scratch is that they dont come with accessibility, theyre not tested etc so in the long run its more costly.

Reply:
i have made the necessary changes